Node/samples/app.ts

```diff
@@ -148,6 +151,19 @@ bot.dialog('FetchMemberList', function (session: builder.Session) {
   );
 });
 
+bot.dialog('FetchConversationMember', function (session) {
+  var conversationId = session.message.address.conversation.id;
+  var memberId = session.message.user.id;
+  connector.fetchMember((<builder.IChatConnectorAddress>session.message.address).serviceUrl, conversationId, memberId, function (err, result) {
+      if (err) {
```

Comment:
Use arrow function `(err, result) => { ... }` instead of `function (err, result) { ... }`